.github/workflows/pipeline-deploy-app-int.yaml

```diff
@@ -0,0 +1,40 @@
+name: Pipeline Deployment Application Integration Environment
+
+on:
+  workflow_call:
+    inputs:
+      workspace:
+        description: "The workspace to deploy the service into"
+        required: true
+        type: string
+      commit_hash:
+        description: "The commit hash to identify the version of artefacts"
+        required: true
+        type: string
+
+jobs:
+
+  deploy-to-integration:
+    name: Deploy to integration environment
+
+    uses: NHSDigital/uec-dos-management/.github/workflows/build-full-application.yaml@v0.35
+    with:
+      environment: int
+      workspace: ${{ inputs.workspace }}
+      triggering_repo: uec-dos-service-management
+      triggering_repo_commit_hash: ${{ inputs.commit_hash }}
+      multi_domain: 'true'
+    secrets: inherit
+
+  # integration-tests:
+  #   name: Run integration tests
+  #   needs:
+  #     [
+  #       deploy-application-infrastructure,
+  #       deploy-application,
+  #     ]
+  #   uses: NHSDigital/uec-dos-management/.github/workflows/integration-test.yaml@v0.35
+  #   with:
+  #     environment: test
```

Comment:
I understand it's commented out but worth change this to 'int'?

Reply:
Decided to remove and replace with a TODO as this will probably take a different format anyway